### PR TITLE
Feature/daily send text

### PR DESCRIPTION
**Pull request by meganrm**
This is the file we will schedule to run once a day. Right now it's just sending dummy data

### CODE REVIEW

File: models/user.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = class User {
     let updates = {};
     req.session.districts.forEach(district => {
       let path = `sms-users/${district.state}/${district.district}/`;
-      let newPostKey = firebasedb.ref(path).push().key;
+      let newPostKey = `${this.phoneNumber}-${this.zipcode}`;
       updates[path + newPostKey] = this;
     });
```
Comment (meganrm) [inline models/user.js]: I changed this so we dont have duplicate people. There can only be one user with any phone number and zip combo

**Comment by DarDavis** [comment]
One of the tests failed on this PR

File: models/user.js (hunk 1 above)
Comment (DarDavis) [reply models/user.js]: So the key is just their phoneNum and Zip now

Comment (DarDavis) [reply models/user.js]: It seems Travis is having a hard time spinning up our server

**Comment by meganrm** [comment]
@DarDavis i fixed it, now can you review?

**Pull request merged**